styles/style.css

```diff
@@ -6,3 +6,178 @@ Green: #00B172
 White: #FFF
 
 */
+
+
+html,
+body{
+    margin:0;
+    padding:0;
+    font-size: 10px;
+    font-family: Helvetica, sans-serif
+}
+
+
+
+.nav-bar{
+    width: 100%;
+  padding-top: 14px;
+  padding-bottom: 14px;
+  display:flex;
+  align-items: center;
+  position: fixed;
+  background-color: white;
+  justify-content: space-between;
+  height: 65px;
+  padding: 0 20px;
+  box-sizing: border-box;
+}
+
+
+
+.greenlogo{
+  height: 35px;
+
+}
+
+.nav-bar-ul{
+    display: flex;
+    margin: 0 auto;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  align-content: center; 
+}
+
+.nav-bar-ul li{
+list-style-type: none;
+  text-align: center;
+  padding: 0px 30px;
+}
+
+.nav-bar a{
+text-decoration: none;
+  color: #1A1A1A;
+  font-size: 2em
+}
+
+header{
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+h1{
+  font-size: 4em;
+  color: white;
+  }
+
+h2{
+    font-size: 2.5em;
+}
+
+h3{
+    color: #00B172;
+    font-weight: bold;
+}
+
+h5{
+    font-size: 2em;
+}
+
+
+header{
+  background-image: url("../images/landing.jpg");
+  background-size: cover;
+  background-position: bottom;
+  padding-top: 50px;
+  height: 75vh;
+  color: #ffffff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 600px;
+}
+
+header{
+    height: 650px;
+  }
+
+  header p{
+      font-size: 20px;
+      text-align: center;
+      width: 70%;
+      font-weight: lighter;
+  }
+  
+  .section2{
+    text-align: center;
+    padding-bottom: 50px;
+  }
+  
+  .section2 div{
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    flex-flow: row;
+    align-items: baseline;
+    align-content: center;
+  }
+
+  .blok{
+      font-size: 1.5em;
+      text-align: center;
+      box-sizing: border-box;
+      padding: 20px;
+      margin: 50 5% 0 5%;
+      width: 40%;
```

Comment:
Since you have three articles, it would flow better if you changed line 133 to `width:33%;`